### PR TITLE
Biome + flesh out useUtils

### PR DESCRIPTION
**Pull request by Brendonovich**
Adds biome formatting and moves `useUtils` to `@rspc/query-core`, fleshing it out with more methods.

### CODE REVIEW

File: packages/query-core/package.json [hunk 1]
```diff
@@ -5,37 +5,36 @@
...
 	"scripts": {
-		"dev": "rollup -c -w",
-		"build": "rollup -c",
-		"prepare": "rollup -c"
+		"dev": "tsup --watch",
+		"build": "tsup",
+		"prepare": "tsup"
 	},
 	"dependencies": {
-		"@rspc/client": "workspace:*"
```
Comment (oscartbeaumont) [inline packages/query-core/package.json]: Does this not need to exist? It should probs be a peer dep though?

Comment (Brendonovich) [reply packages/query-core/package.json]: I'm not sure, solid-query has query-core as a direct dependency, which feels similar to this

**Comment by Mau-MD** [comment]
CleanShot 2024-07-01 at 19 46 49
This might have broken prod. Cannot build

**Comment by Mau-MD** [comment]
downgrading to 0.2.0 fixed the problem

**Pull request merged**